guides/how-to-integrate-segment-with-novu.mdx

```diff
@@ -0,0 +1,238 @@
+
+## Setting Novu as a Destination
+
+As of <Tooltip tip="November 16th, 2023">taday</Tooltip> Segment doesn’t have a native Novu Destination integration **yet** .
+
+We will be using [Destination Functions](https://segment.com/docs/connections/functions/destination-functions/). Destination functions allow you to transform and annotate your Segment events and send them to any external tool or API without worrying about setting up or maintaining any infrastructure.
+
+### Create a destination function
+- From your workspace, go to `Connections > Catalog` and click the `Functions` tab.
+- Click `New Function`.
+- Select `Destination` as the function type and click `Build`.
+- After you click `Build`, a code editor appears. Use the editor to write the code for your function, configure settings, and test the function’s behavior.
+
+<Tip>Want to see some example functions? Check out the templates available in the Functions UI, or in the open-source Segment Functions Library.</Tip>
+
+### Code the destination function
+Segment invokes a separate part of the function (called a “handler”) for each event type that you send to your destination function.
+
+<Note> Your function isn’t invoked for an event if you’ve configured a destination filter, and the event doesn’t pass the filter. </Note>
+
+The default source code template includes handlers for all event types. You don’t need to implement all of them - just use the ones you need, and skip the ones you don’t.
+
+Destination functions can define handlers for each message type in the Segment spec:
+
+- onIdentify
+- onTrack
+- onPage
+- onScreen
+- onGroup
+- onAlias
+- onDelete
+- onBatch
+
+Each of the functions above accepts two arguments:
+
+`event` - Segment event object, where fields and values depend on the event type. 
+For example, in “Identify” events, Segment formats the object to match the Identify spec.
+
+`settings` - Set of settings for this function.
+The example below shows a destination function that listens for “Track” events, and sends some details about them to an external service.
+
+```javascript
+async function onTrack(event) {
+  await fetch('https://example-service.com/api', {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json'
+    },
+    body: JSON.stringify({
+      event_name: event.event,
+      event_properties: event.properties,
+      timestamp: event.timestamp
+    })
+  })
+}
+```
+To change which event type the handler listens to, you can rename it to the name of the message type. 
+
+For example, if you rename this function `onIdentify`, it listens for “Identify” events instead.
+
+### Sending a payload to Novu
+
+Now let's learn how to send a payload to Novu based on the `onTrack` event function:
+
+To successfully send a payload to Novu, triggering the appropriate workflow and delivering it to the correct customer, please ensure that you provide the following information:
+
+1. Novu's Workflow Identifier.
```

Comment:
Can you link this to the workflow section or the workflow concept section?

Reply:
Done.